checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -986,38 +997,38 @@ private String receiverAsString(FieldAccessNode fieldAccessNode) {
   }
 
   /**
-   * Finds the argument passed in the {@code @MustCallAlias} position for a call.
+   * Finds the argument(s) passed in the {@code @MustCallAlias} position(s) for a call.
```

Comment:
No need to write "argument(s)" and "position(s)", you can just write "arguments" and "positions."  It should be clear that if there are no such arguments / positions, the method returns an empty list.